functional/FunctionalTest.java

```diff
@@ -2511,7 +2556,8 @@ public static void copyObject_test12() throws Exception {
       ServerSideEncryption sse = ServerSideEncryption.withManagedKeys("keyId", myContext);
 
       try (final InputStream is = new ContentInputStream(1)) {
-        client.putObject(bucketName, objectName, is, 1, sse);
+        client.putObject(bucketName, objectName, new BufferedInputStream(is), Long.valueOf(1), null,
```

Comment:
How about `1L`?

Reply:
The compiler internally converts `1L` to `Long.valueOf(1)` so IMO this should be kept as it is.

Reply:
If so, why we need to use `Long.valueOf()` i.e. converting default primitive number type to long.  `L` suffix is much better.

Reply:
done.